nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,110 @@
+| **TeamsJs** | Teams Java script client library enables developers to create app content hosted in [IFrame](https://developer.mozilla.org/docs/Web/HTML/Element/iframe) across the Microsoft 365 ecosystem | [TeamsJS ](../docs-ref-autogen/%40microsoft/teams-js/index.yml) |
+| **TeamsFx** | TeamsFx helps to reduce tasks by using Microsoft Teams single sign-on (SSO) and accessing cloud resources down to single line statements with little configuration | [Teamsfx](../docs-ref-autogen/%40microsoft/teamsfx/index.yml) |
 
-## Finding the library
+## Microsoft Teams Live Share
 
-The TeamsJS client is distributed as an npm package. The latest version can be found here:
+The [Live Share SDK](https://github.com/microsoft/live-share-sdk) is compromised of three sub-packages with features to help developers create more collaborative moments across Microsoft Teams. Live Share (or sometimes called Live Share core) is for connecting to Fluid Framework through `sidePanel` and `meetingStage` for in meeting pop-up cards. Live Share media allows for media synchronization for HTML `<video>` and `<audio>` elements; which are beneficial for presenting video and audio is a Teams session. Live Share canvas adds annotation tools for whiteboard and PowerPoint presentations. 
 
-www.npmjs.com/package/@microsoft/teams-js
+### Finding the library
 
-## Installing the library
+The Live Share SDK is a JavaScript package comprised of three sub-packages ([Live Share](/msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md), [Live Share media](/msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-media-capabilities.md), and [Live Share canvas](/msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md)) which are published on npm and can be installed using npm or yarn.
 
-You can install the package using npm or yarn:
+To install the latest version of the Live Share packages to your application:
 
-* `npm install --save @microsoft/teams-js`
-* `yarn add @microsoft/teams-js`
+#### Live Share core
 
-## Using the library
+# [NPM](#tab/npm)
 
-If you are using any dependency loader or module bundler such as [RequireJS](http://requirejs.org/), [SystemJS](https://github.com/systemjs/systemjs), [browserify](http://browserify.org/), or [webpack](https://webpack.github.io/), you can use `import` syntax to import specific modules. For example:
+```bash
+npm install @microsoft/live-share@next --save
+```
+# [Yarn](#tab/yarn)
+
+To install using yarn
+```bash
+yarn add @microsoft/live-share@next
+```
+
+---
+
+#### Live Share media
 
```

Comment:
Add brief intro here. Consider using [this description from docs](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/teams-live-share-media-capabilitie):

The Live Share SDK enables robust media synchronization for any HTML <video> and <audio> element with just a few lines of code. By synchronizing media at the player state and transport controls layer, you can individually attribute views and license, while providing the highest possible quality available through your app.